subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/TestDescriptor.java

```diff
@@ -41,6 +42,15 @@
     @Nullable
     String getClassName();
 
+    /**
+     * The name for display. It may be the same as or different from {@link #getName()}
+     *
+     * @return the name for display.
```

Comment:
:o: The wording is a little inconsistent with the rest of the class.

Reply:
Fixed